### ISSUE TITLE
Read-Only public dashboard list

### ISSUE THREAD

**Issue by ComputerTech312** [louislam/uptime-kuma#1839]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Making the panel public but not editable by users not logged in.
I think the panel should have a option to allow users not logged in to be able to view but not edit or change anything.

### ✔️ Solution

Add the option for view only.

**Comment by kristiandg** [comment]
Agreed - this would be a crucial item to publishing a public-facing dashboard for "customers" (look, but don't touch). ;)

**Comment by CommanderStorm** [comment]
@ComputerTech312
I think this feature is better covered by using status pages.
Could you go into details why this would be not an option for you?

**Comment by melroy89** [comment]
While I think the status page make sense indeed, the current status page is very limited in terms of details. Like you can't actually click on the service to get more details about it.

@CommanderStorm  Just like https://status.uptimerobot.com/, I would expect clicking on a service would bring you to a page with a bigger status uptime bar, overall uptime statistics, response times graph as well as a recent events.

In fact, you already built all these features, but for some reason it's not available in the status page 😢

**Comment by CommanderStorm** [comment]
Those enhancements are tracked in other issues though..

This issue specifically requests to make the dashboard list read only public.

Given that the rationale behind this issue does not seem clear (and quite close to other better worded issues like Multi-User auth, ...) I am going to close this in favor of the other issues.

**Comment by melroy89** [comment]
Ok, I understand. thanks!
